Dockerfile

```diff
@@ -30,14 +33,16 @@ RUN yum -y install \
     luasandbox-parquet* \
     luasandbox-rjson* \
     parquet-cpp* \
-    && yum clean all
+    && dnf clean all
 
+WORKDIR /app
 COPY . /app
+RUN pip3 install -r requirements.txt
+RUN mvn dependency:copy-dependencies
 
-RUN rm -fr release/; mkdir release/
-
+RUN rm -fr /app/release; mkdir /app/release
 WORKDIR /app/release
+RUN cmake ..; make
 
-RUN ln -s /usr/bin/cmake3 /usr/bin/cmake; cmake ..; make
-
-CMD ctest3 -V -C hindsight
```

Comment:
Is this PR removing all use of hindsight? If so, we should remove the dependencies in this file too so that we no longer pull in the hindsight libraries.

Reply:
I think removing hindsight + parquet removal should come in a separate PR that also introduces a GRAVEYARD.md.